lib/Zadm/Schema/base.pm

```diff
@@ -27,22 +27,22 @@ has schema => sub($self) {
     bootargs    => {
         optional    => 1,
         description => 'boot arguments for zone',
-        validator   => sub { return undef },
+        validator   => $self->sv->regexp(qr/^.*$/, 'expected a string'),
     },
     pool        => {
         optional    => 1,
         description => 'name of the resource pool this zone must be bound to',
-        validator   => sub { return undef },
+        validator   => $self->sv->regexp(qr/^.*$/, 'expected a string'),
     },
     limitpriv   => {
         description => 'the maximum set of privileges any process in this zone can obtain',
         default     => 'default',
-        validator   => $self->sv->regexp(qr/^[-+!\w,]+$/, 'limitpriv not valid'),
+        validator   => $self->sv->regexp(qr/^[-+!\w,]*$/, 'limitpriv not valid'),
     },
     brand       => {
         description => "the zone's brand type",
         default     => 'lipkg',
-        validator   => sub { return undef },
+        validator   => $self->sv->regexp(qr/^.+$/, 'expected a string'),
```

Comment:
brand names consisting of spaces ?

Reply:
the brand names are checked when `zadm` gets called. this has just been changed for consistency with other validators